corehq/apps/analytics/static/analytix/js/hubspot.js

```diff
@@ -91,7 +91,7 @@ hqDefine('analytix/js/hubspot', [
                 });
             },
             onFormSubmit: function ($form) {
-                $('#get-demo-cta-calendar-content').toggleClass("hidden");
+                $('#get-demo-cta-calendar-content').fadeIn();
```

Comment:
Why is this inconsistent with the line below? This seems easy for someone to innocently re-break in the future.

Reply:
Both the forms(Request Trial and Request Demo) have the same functions exposed so accidentally "Request Demo" form was edited while I changed the "Trail Form", I just reverted it how it was before.

Should I make changes to the demo form also, which I implemented in the trial form?

Reply:
Yeah, I think so. But no need to do it in this PR if you want to get the bugfix out.